app/presenters/detailed_guide_presenter.rb

```diff
@@ -0,0 +1,81 @@
+class DetailedGuidePresenter < ContentItemPresenter
+  include ExtractsHeadings
+  include ActionView::Helpers::UrlHelper
+
+  def body
+    content_item["details"]["body"]
+  end
+
+  def contents
+    extract_headings_with_ids(body).map do |heading|
+      link_to(heading[:text], "##{heading[:id]}")
+    end
+  end
+
+  def context
+    parent["title"]
+  end
+
+  def breadcrumbs
+    [
+      { title: "Home", url: "/" },
+      { title: context, url: parent["base_path"] }
+    ]
+  end
+
+  def from
+    links("lead_organisations") + links("supporting_organisations") + links("worldwide_organisations")
```

Comment:
Is Case study effectively the same? If they are then the content items should match. Thoughts @boffbowsh?